core/storage/config/gae_models.py

```diff
@@ -180,3 +180,67 @@ def create(
             rules=rule_dicts,
             rule_schema_version=rule_schema_version,
             default_value=default_value)
+
+
+class FeatureFlagModel(base_models.BaseModel):
+    """A class that represents named dynamic feature-flag.
+    This model only stores fields that can be updated in run time.
+
+    The id is the name of the feature-flag.
+    """
+
+    force_enable_for_all_users = datastore_services.BooleanProperty(
+        default=False)
+    rollout_percentage = datastore_services.IntegerProperty(default=0)
+    user_group_ids = datastore_services.JsonProperty(default=[])
```

Comment:
I don't think you should specify a default for JsonProperty like this (there are gotchas with passing [] or {} as default args in Python), and I think this is wrong anyway. It should be a StringProperty with repeated=True.

Reply:
I have updated this.